geps/gep-1911.md

```diff
@@ -0,0 +1,238 @@
+# GEP-1911: Backend Protocol Selection
+
+* Issue: [#1911](https://github.com/kubernetes-sigs/gateway-api/issues/1911)
+* Status: Provisional
+
+(See status definitions [here](overview.md#status).)
+
+## TLDR
+
+Not all implementations support automatic protocol selection. Even in some cases protocols are disabled without an explicit opt-in (eg. websockets). Thus application developers need the ability to specify the protocol(s) that their application supports.
```

Comment:
> Even in some cases protocols are disabled without an explicit opt-in (eg. websockets)

What do you mean here ? can you give an example?

Reply:
Contour by default when using their HTTPProxy resource doesn't support websockets out of the box. You need to set a bool attribute to true - details here: https://projectcontour.io/docs/1.25/config/websockets/

Reply:
Mentioned nginx and contour in this tl;dr where the websocket protocols were off by default